File: SurgeonPortal.Api/ClientApp/src/app/api/models/medicaltraining/user-certificate-read-only.model.ts

export interface IUserCertificateReadOnlyModel {
    certificateId: number;
    userId: number;
    documentId: number;
    certificateTypeId: number;
    certificateType: string;
    issueDate: string;
    certificateNumber: string;
    documentName: string;
    uploadDateUtc: string;
}

File: SurgeonPortal.Models/MedicalTraining/UserCertificateReadOnlyModel.cs
using System;

namespace SurgeonPortal.Models.MedicalTraining
{
    public class UserCertificateReadOnlyModel
    {
        public int CertificateId { get; set; }
        public int UserId { get; set; }
        public int DocumentId { get; set; }
        public int CertificateTypeId { get; set; }
        public string CertificateType { get; set; }
        public DateTime IssueDate { get; set; }
        public string CertificateNumber { get; set; }
        public string DocumentName { get; set; }
        public DateTime UploadDateUtc { get; set; }
    }
}
